frontend/packages/dev-console/src/components/formik-fields/DropdownField.tsx

```diff
@@ -32,7 +57,9 @@ const DropdownField: React.FC<DropdownFieldProps> = ({ label, helpText, required
           setFieldValue(props.name, value);
           setFieldTouched(props.name, true);
         }}
+        isValid={!isDuplicateEntry}
       />
+      {isDuplicateEntry && <p className="odc-dropdown-field__inline-error">Duplicate detected.</p>}
```

Comment:
No need for this because the `FormGroup` just above already has `isValid` and `errorMessage` props.